languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/.meta/example.lisp

```diff
@@ -0,0 +1,24 @@
+(defpackage :lillys-lasagna-leftovers
+  (:use :cl)
+  (:export
+   :preparation-time
+   :remaining-minutes-in-oven
+   :split-leftovers))
+
+(in-package :lillys-lasagna-leftovers)
+
+(defun preparation-time (&rest layers) (* 19 (length layers)))
+
+(defun remaining-minutes-in-oven (&optional (length :normal))
+  (+ 337 (if (not length) -337
+             (case length
+               (:longer 100)
+               (:very-long 200)
+               (:shorter -100)
+               (:very-short -200)
+               (t 0)))))
```

Comment:
Should we allow other parameters? I really don't think it matters all too much, so we could keep this or use `ecase` probably if we don't want to accept other lengths

Reply:
The way i constructed the exercise the default (with no argument given) should use this value.

Reply:
Haha, duh, sorry! I missed that! That looks good then :)